examples/README.md

```diff
@@ -0,0 +1,26 @@
+# SigOpt API Examples
+For the complete API documentation, visit [https://sigopt.com/docs](https://sigopt.com/docs).
+## Python
+Check out our `basic` python example and implement your own function to evaluate. Dive deeper with our `parallel` example, which runs multiple optimization loops at the same time.
```

Comment:
Gotcha, I thought these were pointers to the docs. Makes sense. Thanks!

On Tue, Feb 16, 2016 at 11:51 AM, Alexandra L Johnson <
notifications@github.com> wrote:

> In examples/README.md
> <https://github.com/sigopt/sigopt-python/pull/29#discussion_r53067802>:
>
> > @@ -0,0 +1,26 @@
> > +# SigOpt API Examples
> > +For the complete API documentation, visit [https://sigopt.com/docs](https://sigopt.com/docs).
> > +## Python
> > +Check out our `basic` python example and implement your own function to evaluate. Dive deeper with our `parallel` example, which runs multiple optimization loops at the same time.
>
> This is the screen view they'll see when they're viewing the readme, so I
> hope it'll be clear which examples we're referring to. If this gets more
> complicated we can add links but I think people who reach this point will
> feel comfortable diving in.
> [image: screen shot 2016-02-16 at 11 50 33 am]
> <https://cloud.githubusercontent.com/assets/913838/13089000/8cd202d0-d4a3-11e5-8ec2-6558273016d6.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sigopt/sigopt-python/pull/29/files#r53067802>.
>



-- 
Scott Clark
Co-founder and CEO
SigOpt, Inc. - www.sigopt.com - @SigOpt
scott@sigopt.com - @DrScottClark